src/rt/backtrace/dwarf.d

```diff
@@ -15,6 +15,7 @@ module rt.backtrace.dwarf;
 version(CRuntime_Glibc) version = glibc_or_bsdlibc;
 else version(FreeBSD) version = glibc_or_bsdlibc;
 else version(DragonFlyBSD) version = glibc_or_bsdlibc;
+else version(CRuntime_UClibc) version = glibc_or_bsdlibc;
```

Comment:
Think about renaming this to something like `has_backtrace`.

Reply:
Could be done in another PR. Like to keep this simple and focused on adding the new defs.

Reply:
Would rather you change it now, nobody ever comes back and changes this stuff later. Renaming a single variable hardly makes this pull complex. :smile:

Reply:
OK, renamed it.